Makefile

```diff
@@ -5,21 +5,20 @@ include Makefile.envs
 .PHONY=check
 
 FILEPACKAGER=$$EM_DIR/tools/file_packager.py
-UGLIFYJS=npx uglifyjs
-PRETTIER=npx prettier
```

Comment:
Thanks a lot @pmp-p ! Could you please merge upstream/main in as I think this was added just a few commits ago on main.